schema.sql

```diff
@@ -1,13 +1,19 @@
--- schema.sql
+-- Create the 'owners' table
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INT
+);
 
-CREATE TABLE animals (
+-- Create the 'species' table
+CREATE TABLE species (
     id SERIAL PRIMARY KEY,
-    name VARCHAR(255),
-    date_of_birth DATE,
-    escape_attempts INT,
-    neutered BOOLEAN,
-    weight_kg DECIMAL(5, 2)
+    name VARCHAR(255)
 );
 
--- Add the 'species' column
-ALTER TABLE animals ADD COLUMN species VARCHAR(255);
```

Comment:
- [ ] [OPTIONAL]  Great job on removing the command for adding the species column, however, it would be better use the ALTER TABLE statement to Remove column `species`, please kindly do that. For Example:
`ALTER TABLE animals ADD COLUMN species VARCHAR(255);
`
`ALTER TABLE animals DROP column species;`